### PR TITLE
DROOLS-5011 : Multiselect dropdown for enumerated values

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.redhat.com/browse/DROOLS-5011

https://github.com/kiegroup/kie-wb-common/pull/3142
https://github.com/kiegroup/drools-wb/pull/1301

### CODE REVIEW

**Comment by Rikkola** [comment]
@jomarko This PR fixes the issue. kie-wb-common PR is an additional fix that allows the real value text and not the actual value to show up in the template editor grid. I'm still writing tests, but the fix itself should work. Running these PRs as drafts to see if I broke any existing tests.

File: drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-client/src/main/java/org/drools/workbench/screens/guided/template/client/editor/TemplateDataCellValueFactory.java [hunk 1]
```diff
@@ -314,7 +314,7 @@ public String convertToModelCell(TemplateDataColumn column,
         DataType.DataTypes dataType = getDataType(column);
 
         if (OperatorsOracle.operatorRequiresList(column.getOperator())) {
-            return cv.getValue().toString();
+            return (cv.getValue() == null ? null : cv.getValue().toString());
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-client/src/main/java/org/drools/workbench/screens/guided/template/client/editor/TemplateDataCellValueFactory.java]: @jomarko Ok this null check was missing. I checked return values for other column types and looks like it is normal that the cv.getValue() can be null.

**Comment by Rikkola** [comment]
I still need to write tests, but will do that later today.

**Comment by Rikkola** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
Failed tests:
```
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.checkTypeSingleCompositeWithNestedCompositeCollection
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.retrieveFactModelTupleDmnList
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.createTopLevelFactModelTreeCompositeCollection
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.checkTypeSingleCompositeWithCollection
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.createTopLevelFactModelTreeCompositeNoCollection
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.checkTypeRecursive
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.createTopLevelFactModelTreeRecursiveTypes
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.retrieveFactModelTupleDmnListComposite
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.initializeNameAndNamespace
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.createTopLevelFactModelTreeSimpleNoCollection
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.createTopLevelFactModelTreeSimpleCollection
org.drools.workbench.screens.scenariosimulation.backend.server.DMNTypeServiceImplTest.checkTypeSimpleTopLevelCollection
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.addToScenarioRecursive
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.findNewIndexOfGroup
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.createSettings
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.createSimulationCornerCases
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.createBackground
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.createSimulation
org.drools.workbench.screens.scenariosimulation.backend.server.util.DMNSimulationSettingsCreationStrategyTest.addEmptyColumnIfNeeded
```

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by yesamer** [comment]
jenkins retest this please

**Comment by yesamer** [comment]
@manstis ok, I think it can be merge now! :)

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Could I get a merge for these 2 PRs thank you.

**Pull request merged**